frontend/src/components/Navbar.js

```diff
@@ -1,12 +1,143 @@
 import React from 'react';
 import Logout from '../components/Logout';
+import { useCookies } from 'react-cookie';
+import { Link } from 'react-router-dom';
 const Navbar = () => {
+  const [user] = useCookies(['user']);
   return (
-    <div>
-      <h1>
-        Navbar will be <Logout />
-      </h1>
-    </div>
+    <nav className="bg-white border-gray-200 px-2 sm:px-4 py-2.5 rounded dark:bg-gray-800">
+      <div className="container flex flex-wrap justify-between items-center mx-auto">
+        <a href="#" className="flex items-center">
+          <img
+            src="https://res.cloudinary.com/dxqrl41la/image/upload/v1647070109/Logo/logo_vdr5hs.png"
+            className="mr-3 h-6 sm:h-10"
+            alt="Flowbite Logo"
+          />
+        </a>
+        <div className="flex items-center md:order-2">
+          <button
+            type="button"
+            className="flex mr-3 text-sm bg-gray-800 rounded-full md:mr-0 focus:ring-4 focus:ring-gray-300 dark:focus:ring-gray-600"
+            id="user-menu-button"
+            aria-expanded="false"
+            data-dropdown-toggle="dropdown">
+            <span className="sr-only">Open user menu</span>
+            <img
+              className="w-8 h-8 rounded-full"
+              src="https://images.unsplash.com/photo-1506794778202-cad84cf45f1d?ixlib=rb-1.2.1&ixid=MnwxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHx8&auto=format&fit=crop&w=687&q=80"
+              alt="user photo"
+            />
+          </button>
+          <div
+            className="hidden z-50 my-4 text-base list-none bg-white rounded divide-y divide-gray-100 shadow dark:bg-gray-700 dark:divide-gray-600"
+            id="dropdown">
+            <div className="py-3 px-4">
+              <span className="block text-sm font-medium text-gray-500 truncate dark:text-gray-400"></span>
+            </div>
+            <ul className="py-1" aria-labelledby="dropdown">
+              <li>
```

Comment:
instead of creating new `<li>` element you can create something like

```
const navbarDropdownConfiguration = [
    { id: 1, text: 'Dashboard', value: 'hrefValue' },
    { id: 2, text: 'Settings', value: 'hrefValue' },
    { ... },
]
```

and then map through it:

```
{navbarDropdownConfiguration.map(({ id, text, value }) => (
              <li key={id}>
                <a
                  href={value}
                  className="block py-2 px-4 text-sm text-gray-700 hover:bg-gray-100 dark:hover:bg-gray-600 dark:text-gray-200 dark:hover:text-white">
                  {text}
                </a>
              </li>
      ))}
```